recipes/cyclus/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+
+if [ "${UNAME}" == "Darwin" ]; then
```

Comment:
I am not sure that this works and maybe be the problem on `OS X` can you try:

`if [ "$(uname)" == "Darwin" ]; then` instead?